### PR TITLE
WIP Android 5

### PR DESCRIPTION
**Pull request by mbernasocchi**
!!!Don't merge, for review only!!!

### CODE REVIEW

File: android/AndroidManifest.xml [hunk 1]
```diff
@@ -74,7 +74,7 @@
     <activity android:name="ch.opengis.qfield.QFieldAppRaterActivity" />
 
   </application>
-  <uses-sdk android:minSdkVersion="23" android:targetSdkVersion="26" />
+  <uses-sdk android:minSdkVersion="21" android:maxSdkVersion="22"  android:targetSdkVersion="26" />
```
Comment (signedav) [inline android/AndroidManifest.xml]: Well I'm not familiar with it, so I'm just thinking. Does it make sense to have the targetSdkVersion higher than the max?

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-512-7163f85106f3aab0ac42568f7c1936ccea2bf53e-armv7.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-512-7163f85106f3aab0ac42568f7c1936ccea2bf53e-x86.apk)

File: android/AndroidManifest.xml (hunk 1 above)
Comment (mbernasocchi) [reply android/AndroidManifest.xml]: it is a requirement by the play store. no way around it

Comment (mbernasocchi) [reply android/AndroidManifest.xml]: FYI: https://developer.android.com/distribute/best-practices/develop/target-sdk

**Comment by signedav** [comment]
Testing on device was good. Is there something special I need to put an eye on?

**Comment by mbernasocchi** [comment]
opening projects, adding favourite folders by long clicking a folder, removing favourites by longclicking. 
and rating system with `adb shell am start -n ch.opengis.qfield/.QFieldActivity --el LaunchCount 1 --el FirstLaunch 1546300800 --ez DontShowAgain false` then close qfield and reopen it from android itself, now you should get prompted to rate

File: android/AndroidManifest.xml (hunk 1 above)
Comment (signedav) [reply android/AndroidManifest.xml]: Easypeasy

**Comment by signedav** [comment]
Looks good :+1:

**Comment by signedav** [comment]
Screenshot_2019-03-12-15-31-51
Screenshot_2019-03-12-15-30-59

**Comment by mbernasocchi** [comment]
great thanks

**Comment by qfield-fairy** [comment]
Uploaded test apks for [armv7](https://download.opengis.ch/qfield/ci-builds/qfield-dev-512-12b5c23049bbe0b8241e58d6ea475446dc91254f-armv7.apk) and [x86](https://download.opengis.ch/qfield/ci-builds/qfield-dev-512-12b5c23049bbe0b8241e58d6ea475446dc91254f-x86.apk)

**Pull request closed without merging**